README.md

```diff
@@ -1,257 +1,30 @@
+🚧 You're looking at the source for `solidus_stripe` v5, which will only support the **starter frontend** and not the classic frontend 🚧
+🚧 You can find the source for [v4 on the corresponding branch](https://github.com/solidusio/solidus_stripe/tree/v4) 🚧
```

Comment:
I'd also add some accurate instructions about what to do to stick to the old version, suggesting to use the Rubygems version, or if someone wants to stick to github, they will need to change the branch.

Reply:
Added and updated the layout:

<img width="974" alt="image" src="https://user-images.githubusercontent.com/1051/210088043-41859c4b-70bf-44ac-a3be-5c404fd4009f.png">

I also added a post install message that should better cover the case of people already using it from source.

Reply:
Lovely, thanks!